.gitignore

```diff
@@ -59,3 +59,6 @@ typings/
 
 # next.js build output
 .next
+
+#package-lock
+package-lock.json
```

Comment:
You should commit your `package-lock.json` file. (npm tells you to commit it as well)

This especially comes into play once you have some CI set up